<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateCoordSystemROI</class>
 <widget class="QDialog" name="DialogCreateCoordSystemROI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="lblY">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblX">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblWidth">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="lblHeight">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbX">
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="sbY">
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sbWidth">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="sbHeight">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="GraphicsViewWithEvents" name="gvOriginal">
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>480</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="gvTransformed">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>480</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsViewWithEvents</class>
   <extends>QGraphicsView</extends>
   <header>graphicsviewwithevents.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCreateCoordSystemROI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogCreateCoordSystemROI</receiver>
   <slot>intValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogCreateCoordSystemROI</receiver>
   <slot>intValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogCreateCoordSystemROI</receiver>
   <slot>intValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogCreateCoordSystemROI</receiver>
   <slot>intValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCreateCoordSystemROI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>intValueChanged(int)</slot>
  <slot>on_buttonBox_accepted()</slot>
 </slots>
</ui>
